### ISSUE TITLE
May be error in MaskIOU module

### ISSUE THREAD

**Issue by AshStuff** [youngwanLEE/centermask2#97]
I am trying to understand MaskIOU module and I find this line confusing while calculating the maskiou targets

https://github.com/youngwanLEE/centermask2/blob/master/centermask/modeling/centermask/mask_head.py#L155

This line thresholds the mask logits. I think the threshold should be on the predicted masks right ? like torch.sigmoid(pred_mask_logits) > 0.5 ?
